imagelab/DisplayImage.java

```diff
@@ -16,7 +17,7 @@
     /** The image of this frame. */
     Image img;
     /** The display panel of this frame. */
-    private DisPanel pane;
+    private DynaPanel pane;
```

Comment:
Using DynaPanel Class instead of DisPanel class so that menu items can still be selected while the music is playing and image is synchronizing.